pkg/config/base.go

```diff
@@ -36,12 +36,13 @@ type BaseConfig struct {
 	WsUrl     string             `yaml:"ws_url"`     // (env LIVEKIT_WS_URL)
 
 	// optional
-	Logging              *logger.Config `yaml:"logging"`                // logging config
-	TemplateBase         string         `yaml:"template_base"`          // custom template base url
-	ClusterID            string         `yaml:"cluster_id"`             // cluster this instance belongs to
-	EnableChromeSandbox  bool           `yaml:"enable_chrome_sandbox"`  // enable Chrome sandbox, requires extra docker configuration
-	MaxUploadQueue       int            `yaml:"max_upload_queue"`       // maximum upload queue size, in minutes
-	DisallowLocalStorage bool           `yaml:"disallow_local_storage"` // require an upload config for all requests
+	Logging                      *logger.Config `yaml:"logging"`                          // logging config
+	TemplateBase                 string         `yaml:"template_base"`                    // custom template base url
+	ClusterID                    string         `yaml:"cluster_id"`                       // cluster this instance belongs to
+	EnableChromeSandbox          bool           `yaml:"enable_chrome_sandbox"`            // enable Chrome sandbox, requires extra docker configuration
+	MaxUploadQueue               int            `yaml:"max_upload_queue"`                 // maximum upload queue size, in minutes
+	DisallowLocalStorage         bool           `yaml:"disallow_local_storage"`           // require an upload config for all requests
+	EnableRoomCompositeSDKSource bool           `yaml:"enable_room_composite_sdk_source"` // attempt to render supported audio only room composite use cases using the SDK source instead of Chrome
```

Comment:
Would prefer this to be automatic, egress config is already pretty extensive

Reply:
Considering there is a not insignificant risk with rolling this out, I think we need a way to enable it on a per cluster basis and to disable it easily. Is there a better way to achieve this?

Reply:
that's fair, can always remove it in the future